flexget/plugins/clients/rtorrent.py

```diff
@@ -139,7 +139,7 @@ def single_request(self, host, handler, request_body, verbose=0):
     def parse_response(self, response):
         p, u = self.getparser()
 
-        response_body = ''
+        response_body = response.read(1024)
```

Comment:
I think this should probably be `response_body = b''` (set it to an empty byte string,) rather than changing the logic here.